count-repeat/meta.json

```diff
@@ -0,0 +1,8 @@
+{
+    "title": "CountRepeat",
+    "name": "countRepeat",
+    "difficulty": "Hard",
+    "author": "Jutulli",
+    "category": "lists"
```

Comment:
should be in the category "string-2"
difficulty should be "medium"
function names should follow python snake case, so here it should be name: count_repeat (with quotes). you need to change the starter.py as well

Reply:
Whoops, everything should be fixed now